src/commands/manifest/cmd-manifest-auto.ts

```diff
@@ -110,6 +111,22 @@ async function run(
     return
   }
 
+  if (existsSync(path.join(dir, 'environment.yml'))) {
```

Comment:
@pvdz We should also check for `.yaml` too?

Reply:
sure!